### ISSUE TITLE
Nice animation: Evolution of AlphaFS (Gource Visualization)

### ISSUE THREAD

**Issue by Yomodo** [alphaleonis/AlphaFS#230]
https://www.youtube.com/watch?v=HPjAwfkF3r8&index=48&list=LL6HfiE9VRG2BxLw6ae6Y8eg

**Comment by alphaleonis** [comment]
Hehe, nice! :)

**Comment by CyberSinh** [comment]
No new commit for one year now. :-(

**Comment by alphaleonis** [comment]
Yeah, I guess time is the problem for us as always. But of course anyone is welcome to help out and submit a PR or two.

**Comment by CyberSinh** [comment]
In the .NET Framework 4.6.2 runing under Windows 10 Redstone, it is now possible to allow native long paths support: https://blogs.msdn.microsoft.com/dotnet/2016/08/02/announcing-net-framework-4-6-2/#bcl

What do you think about the Microsoft solution VS the AlphaFS implementation (compatibility, performance, etc.)?

Thanks.
